packages/flutter_tools/test/general.shard/runner/flutter_command_test.dart

```diff
@@ -52,7 +52,8 @@ void main() {
     testUsingContext('honors shouldUpdateCache true', () async {
       final DummyFlutterCommand flutterCommand = DummyFlutterCommand(shouldUpdateCache: true);
       await flutterCommand.run();
-      verify(cache.updateAll(any)).called(1);
+      // First call for universal, second for the rest
+      verify(cache.updateAll(any)).called(2);
```

Comment:
You could verify that each call made sense with `captureAny`

Reply:
cool, didn't know about `captureAny`. Done.